numba/core/transforms.py

```diff
@@ -211,13 +212,55 @@ def _loop_lift_modify_blocks(func_ir, loopinfo, blocks,
     return liftedloop
 
 
+def _has_multiple_loop_exits(cfg, lpinfo):
+    """Returns True if there is more than one exit in the loop.
+
+    NOTE: "common exits" refers to the situation where a loop exit has another
+    loop exit as its successor. In that case, we do not need to alter it.
+    """
+    if len(lpinfo.exits) <= 1:
+        return False
+    exits = set(lpinfo.exits)
+    pdom = cfg.post_dominators()
+
+    # Eliminate blocks that have other blocks as post-dominators.
+    processed = set()
+    remain = exits
```

Comment:
```suggestion
    remain = set(exits) # create a copy to work on
```
This should fix the build.

Reply:
Yes, confirmed locally.